PVB Linx Groep 6/Assets/Code/Scripts/UI/C_DamageNotifier1.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+public class C_DamageNotifier : MonoBehaviour
+{
+    [SerializeField] private Text c_DamageText;
+    [SerializeField] private float delayBetweenNotifier_C;
```

Comment:
Rename both variables to no longer contain single letters and underscores, these are only for clarifying access level